pkg/credentialprovider/config.go

```diff
@@ -203,7 +203,7 @@ func ReadURL(url string, client *http.Client, header *http.Header) (body []byte,
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
-		klog.V(2).Infof("body of failing http response: %v", resp.Body)
+		klog.V(2).InfoS("Failing http response", "responseBody", resp.Body)
```

Comment:
I wouldn't log the response body for these errors, things like that have resulted in SSRF issues in the past

Reply:
/sig security
@SergeyKanzhelev  mentions below
> sig security to confirm it is ok to change this to log an actual response body.

Or we may change it to:
```
klog.V(2).InfoS("Failed to read URL", "statusCode", resp.StatusCode, "URL", url)
```
like https://github.com/kubernetes/kubernetes/commit/0c5d9ed0d21f0ebace1b5bd437077bc6f6e070f2#diff-f9f30fa2d7430c70760d157077317b55246295b19a6563437aa81f4a736fcaebR81-R83

https://github.com/kubernetes/kubernetes/blob/0c5d9ed0d21f0ebace1b5bd437077bc6f6e070f2/pkg/credentialprovider/config.go#L80-L85

Reply:
+1 to log status code and url, but not response body